### PR TITLE
Added listen_backlog_multiplier, serialize_accept and memory_cache_enabled to configuration

### PR DESCRIPTION
**Pull request by deu-dev**
**Description of the change**

FIX: black_function_list updated coresponding to PGPOOL Docs
IMP: added listen_backlog_multiplier to pgpool_create_config
IMP: added serialize_accept to pgpool_create_config
IMP: added memory_cache_enabled to pgpool_create_config
IMP: added Misc Configuration Parameters
IMP: added automatic backend node weight based on region

**Benefits**

Now you can configure listen_backlog_multiplier, serialize_accept, memory_cache_enabled and all Misc Configuration Parameters from ENV Variables
Automatic backend node weight setting based on region

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -256,6 +259,14 @@ Pgpool:
...
+- `PGPOOL_RELCACHE_EXPIRE`: Specifies the relation cache expiration time in seconds. Defaults to `0`, which means the cache never expires.
+- `PGPOOL_RELCACHE_SIZE`: Specifies the number of relcache entries. Defaults to `256`.
+- `PGPOOL_RELCACHE_QUERY_TARGET`: The target node to send queries to create relation cache entries ('master', 'load_balance_node'). Defaults to `master`.
+- `PGPOOL_CHECK_TEMP_TABLE`: Setting to catalog or trace, enables the temporary table check in the SELECT statements ('catalog', 'trace', 'none'). Defaults to `catalog`.
+- `PGPOOL_CHECK_UNLOGGED_TABLE`: Setting to on, enables the unlogged table check in the SELECT statements. Defaults to `on`.
+- `PGPOOL_REGION`: Region of the Pgpool-II server. No defaults.
+- `PGPOOL_BACKEND_NODES_REGION_WEIGHT`: Comma separated list of backend nodes region weight in the cluster. No defaults. e.g this value is `pg-0:eu-west:2,pg-1:eu-south:2` and `PGPOOL_REGION` is set to `eu-west` the weight of the backend node `pg-0` will be set to `2` on startup.
```
Comment (dani8art) [inline README.md:269]: Please, add them also to https://github.com/bitnami/bitnami-docker-pgpool#environment-variables section.

**Comment by horiee** [comment]
Can someone merge the PR please?

**Comment by carrodher** [comment]
We are waiting for the feedback to be applied, in the same way, since this PR was created a long time ago, probably a proper rebase will be needed. Would you like to command those changes by creating a new PR implementing those changes but on top of the latest commit present in the `master` branch?

**Comment by horiee** [comment]
@carrodher i'm only interested in mem cache feature.
If that's fine I will take care of that. lmk

**Comment by carrodher** [comment]
Sure, feel free to go ahead! Thanks

**Pull request closed without merging**